syspage.h

```diff
@@ -100,6 +100,9 @@ extern void syspage_addEntries(addr_t start, size_t sz);
 extern int syspage_getMapAttr(const char *map, unsigned int *attr);
 
 
+extern const char *syspage_getMapName(u32 id);
+
+
```

Comment:
Please don't influence on the code implemented by Hubert. The output formatting is his responsonsility.

Reply:
Ok, I'll remove that.

Reply:
My comment is the same as before. Thx for your proposal, but we should strictly preserve roles and responsibilities.

Reply:
Yes, you're right.